models/ProgrammeMembership.ts

```diff
@@ -14,6 +14,8 @@ export interface ProgrammeMembership extends IDateBoxed {
   programmeCompletionDate?: Date;
   curricula: Curriculum[];
   conditionsOfJoining: ConditionsOfJoining;
+  designatedBody?: string;
```

Comment:
Hey hey now, where does this come from, slipping in the side door so quietly? 🔬

Reply:
This whole piece of work is a shoehorning exercise🙈 In my defence, Your Honour, when I added this particular 💩 , I was assuming `designatedBody` would be needed not just DBC. We can always remove it if viewers find this too upsetting? (or pick it up in the next PR?)